zproject/default_settings.py

```diff
@@ -291,6 +291,12 @@
         # 1000 per day per file
         (86400, 1000),
     ],
+    # A zilencer-only limit that applies to requests to the
+    # remote billing system that trigger the sending of an email.
+    "sends_email_by_remote_server": [
+        # 10 emails per day
+        (86400, 10),
```

Comment:
What's the right number? It was originally proposed to do `5`, but that feels kinda too close to what a server where a user or two are playing around with the flow may reasonably run into. So `10` feel like has more margin of error. Maybe even `20` to have confidence that it won't reasonably happen under legitimate use?

Reply:
I think probably 10 is a fine default.